packages/api-gateway/src/drivers/http/server.js

```diff
@@ -18,6 +19,11 @@ const fastify = Fastify({
   logger: !isTestEnv,
 });
 
+// Enable cors
+fastify.register(require('@fastify/cors'), {
+  origin: configCORS.domain,
```

Comment:
I would like to add these options
- **credentials:** this enables to share cookies in requests, I suggest to talk with other teams and know if that's the case this should set to true. [Here](https://stackoverflow.com/questions/24687313/what-exactly-does-the-access-control-allow-credentials-header-do) more information
- **methods:** fastify-cors doesn't say how will behave if this parameter is omitted, so I think we should add it
**allowHeaders:** the default behavior is 
> reflecting the headers specified in the request's Access-Control-Request-Headers header.
[Here](https://github.com/fastify/fastify-cors) you can find more information about it.
Is that what you want?
- **optionSuccessStatus:** to avoid to having conflict with default 204 code with legacy browsers
- **exposedHeaders:** this just if we create custom headers. Do you talk with the rest of cohort if someone is going to need that?

Reply:
thanks for the comments. i will read all the documentation to make a configuration closer to production.

Reply:
I was searching about optionSuccesStatus, I think you can omit that value since almost all browsers accept 204 value (default value). [Here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#browser_compatibility) you can see 204 code compatibility with browsers

Reply:
For the moment I only set the credentials to true to activate the cookies. 
But the other keys I think it is better that they stay like this so as not to put more barriers and interrupt the progress of the other squads.

I don't think it's feasible to change la llave "methds" to list because our requirement is to use only one domain and not several.

Reply:
Ready!